chain/epoch_manager/src/validator_selection.rs

```diff
@@ -684,14 +711,69 @@ mod tests {
         // too low stakes are kicked out
         let kickout = epoch_info.validator_kickout();
         assert_eq!(kickout.len(), 2);
+        #[cfg(feature = "protocol_feature_fix_staking_threshold")]
+        let expected_threshold = 334;
+        #[cfg(not(feature = "protocol_feature_fix_staking_threshold"))]
+        let expected_threshold = 300;
```

Comment:
```suggestion
        let expected_threshold = if cfg!(feature = "protocol_feature_fix_staking_threshold") { 300 } else { 334 }
```

Reply:
I actually wrote it that way first, but I kinda prefer compile time for this